Vagrantfile

```diff
@@ -56,6 +56,7 @@ chmod +x bpf-map
 mv bpf-map /usr/bin
 SCRIPT
 
+$noclean = ENV['NOCLEAN'] ? "export NOCLEAN=1" : ""
```

Comment:
Nice, I think this will speed up the process a lot and thanks for making it default.. Let's see if someone complains.

Reply:
Yep, it sped up provisioning of (powered-off) VMs by ~57% for me :tada: 

I'll switch to make it the default and also change the name accordingly. I'm hesitating between `CLEAN` (bit generic) and `MAKECLEAN` (bit long); what's your opinion?

Reply:
Maybe `MAKECLEAN` seems better. To be fair, one should not need to use `MAKECLEAN` that frequently.